packages/mobile/src/components/Message/Message.component.tsx

```diff
@@ -55,10 +59,13 @@ export const Message: FC<MessageProps> = ({ data }) => {
           </View>
           <View style={{ flexShrink: 1 }}>
             {data.map((message, index) => {
+              const isPendingMessage = pendingMessages[message.id] !== undefined
+              const pendingMessageStyle = isPendingMessage ? classes.pending : {}
+
               const outerDivStyle = index > 0 ? classes.nextMessage : classes.firstMessage
               return (
                 <View style={outerDivStyle} key={index}>
-                  <Typography fontSize={14}>{message.message}</Typography>
+                  <Typography fontSize={14} style={pendingMessageStyle}>{message.message}</Typography>
```

Comment:
For Typography component we use `color` prop so ideally we should avoid adding custom styling ad do it like that instead `color={ pending ? 'grey' : 'default' }`